cdc/api/v2/changefeed.go

```diff
@@ -65,24 +63,24 @@ func (h *OpenAPIV2) CreateChangefeed(c *gin.Context) {
 	}
 	timeoutCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
 	defer cancel()
-	pdClient, err := getPDClient(timeoutCtx, config.PDAddrs, credential)
+	pdClient, err := h.helpers.GetPDClient(timeoutCtx, config.PDAddrs, credential)
 	if err != nil {
 		_ = c.Error(cerror.WrapError(cerror.ErrAPIInvalidParam, err))
 		return
 	}
 	defer pdClient.Close()
 
-	// verify tables
-	kvStorage, err := kv.CreateTiStore(strings.Join(config.PDAddrs, ","), credential)
+	// verify tables todo: del kvstore
+	kvStorage, err := h.helpers.CreateTiStore(config.PDAddrs, credential)
 	if err != nil {
 		_ = c.Error(cerror.WrapError(cerror.ErrInternalServerError, err))
 		return
 	}
 	// We should not close kvStorage since all kvStorage in cdc is the same one.
-	// defer kvStorage.Close()
+	// defer kvStorage.Close() TODO: ?
```

Comment:
TODO: we should use upstream to get a kvStorage instead of call CreateTiStore to get one.